sympy/assumptions/relation/equality.py

```diff
@@ -109,17 +103,182 @@ def negated(self):
     def eval(self, args, assumptions=True):
         return is_neq(*args)
 
-    def _simplify_applied(self, lhs, rhs, **kwargs):
-        eq = Q.eq(lhs, rhs).simplify(**kwargs)
-        return self(*eq.arguments)
 
-    def _eval_binary_symbols(self, lhs, rhs):
-        args = (lhs, rhs)
-        if S.true in args or S.false in args:
-            if lhs.is_Symbol:
-                return {lhs}
-            elif rhs.is_Symbol:
-                return {rhs}
-        return set()
+class StrictGreaterThanPredicate(BinaryRelation):
+    """
+    Binary predicate for $>$.
+
+    The purpose of this class is to provide the instance which represent
+    the ">" predicate in order to allow the logical inference.
+    This class must remain internal to assumptions module and user must
+    use :obj:`~.Gt()` instead to construct the equality expression.
+
+    Evaluation logic of this predicate is delegated to
+    :func:`~.core.relational.is_gt()`
```

Comment:
"Evaluation logic" is probably not intuitive to most people reading this. What you mean is deciding if the inequality is True or False but that could be explained more explicitly.